docs/manage/command_line.rst

```diff
@@ -228,6 +228,17 @@ If you need to permanently delete a Kolibri user and all the data associated wit
 .. warning:: This will permanently erase all the user data.
 
 
+Export Data Logs
+****************
+
+Kolibri database may become large after prolonged use, which can cause the data logs export process from **Facility > Data** the to be slow. Use the following command to export logs from the Terminal or command prompt. 
```

Comment:
```suggestion
During prolonged usage, Kolibri's database increases in size, which slows down the data logs export process from **Facility > Data**. Instead of using the export in the user interface, you can use the following command-line interface to export logs: 
```

Reply:
Do we have an open issue about speed differences between using the UI and the command line? I was not aware of this

Reply:
Me neither - @radinamatic ?

Reply:
The inspiration for documenting came from this [thread on the community forums](https://community.learningequality.org/t/0-13-generating-data-on-session-logs-takes-so-long-doesnt-seem-to-finish/1387).

Reply:
Is the fundamental issue here is that this type of job lacks user feedback and a separate async job process?

Maybe it can be expressed differently:

> The Kolibri user interface currently does not display progress and estimated duration of data log exports. For larger databases, we recommend using the command line interface: (...)

Reply:
Based on the thread in the forums, I don't believe any of the suggestions are accurate.

I filed a new issue here: https://github.com/learningequality/kolibri/issues/6582

If think this is a common problem, we can make a note saying that

> If the log generation appears to hang, we recommend using the command line interface: ...

Otherwise I'd recommend removing this completely

Reply:
I don't see the reason _not_ to document the command. I added the tip in the **Manage > Data** page that links to the command, where the the innacurrate suggestions are removed leaving just the wording about what the user can observe (logs taking long time to generate).

Reply:
This looks good and won't need an update once the issue filed by @indirectlylit is solved. Great work! :100: